stage1/init/common/mount.go

```diff
@@ -45,13 +56,28 @@ func IsMountReadOnly(vol types.Volume, mountPoints []types.MountPoint) bool {
 	return isMPReadOnly(mountPoints, vol.Name)
 }
 
-// GenerateMounts maps MountPoint paths to volumes, returning a list of Mounts.
-func GenerateMounts(ra *schema.RuntimeApp, volumes map[types.ACName]types.Volume) []schema.Mount {
+func convertedFromDocker(ra *schema.RuntimeApp) bool {
+	ann := ra.Annotations
```

Comment:
Right, checking the image's annotation sounds better, since it is a property of the image.